src/plugins/userIndicator/user-indicator.scss

```diff
@@ -0,0 +1,111 @@
+.c-indicator {
```

Comment:
this needs to be updated, just placeholders for styling for now. @charlesh88 to finish this up